packages/matchbox/src/components/Banner/Banner.js

```diff
@@ -95,18 +107,18 @@ const Banner = React.forwardRef(function Banner(props, ref) {
     <StyledContainer
       display="flex"
       flexWrap={['wrap', null, 'nowrap']}
-      p="500"
+      p={size === 'large' ? '500' : '300'}
```

Comment:
`200` here instead of `300` gets closer to the mocks, they are 40px in height in invision, but 50px with `300`

Reply:
Maybe just for `py`, `px` looks fine i think